src/nginx/varify/varify.go

```diff
@@ -54,3 +74,24 @@ func main() {
 		log.Fatalf("Could not write config file: %s", err)
 	}
 }
+
+func readNameServers(resolvConfPath string, defaultNameServer string) ([]string, error) {
+	resolvConfFile, err := os.Open(resolvConfPath)
+	if err != nil {
+		return []string{defaultNameServer}, err
+	}
+	defer resolvConfFile.Close()
+	scanner := bufio.NewScanner(resolvConfFile)
+	var nameServers []string
+	nsRegex := regexp.MustCompile(`^\s*nameserver ([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}).*$`)
```

Comment:
Can we use Go's `net` package to grab the default nameservers, instead of manually parsing `resolv.conf`?

Reply:
Not possible with `net` actually, let's use https://github.com/miekg/dns

Reply:
Lets use: https://github.com/miekg/dns